PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml

```diff
@@ -55,11 +55,13 @@ CellEnergy:                                         # Cell Energy correction com
     enabled: false                                  # Whether to enable the task
     createHistos: false                             # Whether the task should create output histograms
     enableRun2TempCalib: false                      # Whether the temperature calibration should be done for Run2
+    customRecalibFilePath: ""                       # Full path including .root file for custom recalibration object
     cellsNames:                                     # Names of the cells input objects which should be attached to the correction
         - defaultCells                              # This object is defined above in the cells section of the input objects
 CellBadChannel:                                     # Bad channel removal at the cell level component
     enabled: false                                  # Whether to enable the task
     createHistos: false                             # Whether the task should create output histograms
+    customBadChannelFilePath: ""                    # Full path to a custom bad channel OADB object
```

Comment:
Custom bad channels can also be set in the clusterizer (which is why the function is in the correction component base class), so it should also be possible to set that option there

Reply:
As far as i understood, the CellBadChannel part will always be called before the clusterizer. Therefore, the custom bad channel maps should then also be automatically set for the clusterizer if the option is set in CellBadChannel. Please correct me if i'm wrong.

Reply:
While it's true that the bad channel component will tend to be called before the clusterizer, each component loads the request OADB container individually. So with the current implementation, one could load a custom bad channel map for the bad channel component, but then the standard bad channel map for the clusterizer would still be loaded